client/src/routes/modals/ProfileSettingsModal/ProfileSettingsModal.tsx

```diff
@@ -142,6 +150,18 @@ const ProfileSettingsModal = () => {
               onChangeText={setEmail}
               defaultValue={email}
             />
+            {user && !user?.isAnonymous && (
+              <BottomSheetActionTextInput
+                textContentType="currentPassword"
+                secureTextEntry
+                autoCapitalize="none"
+                autoComplete="password-current"
```

Comment:
Oh I've done this too and we got errors in Sentry about it. If I remember correctly this is not available until RN 0.71. I think it needs to be `password` only https://reactnative.dev/docs/0.69/textinput#autocomplete-android

Reply:
aha, so I don't define it for the "newPassword" field I guess?

Reply:
Updated 👍 
I left the new password filed without the autocomplete field